cmd/action/migrate.go

```diff
@@ -49,7 +50,17 @@ var (
 				if err != nil {
 					return err
 				}
-				return migrate.Validate(dir) // TODO(masseelch): tell the user what's wrong
+				if err := migrate.Validate(dir); err != nil {
+					fmt.Fprintf(
+						os.Stderr,
+						"Error: %s\n\nYou have a checksum error in your migration directory.\n"+
+							"This usually happens if you manually create or edit a migration file.\n"+
```

Comment:
usually or can?

Reply:
It will always happen if you manually edit any file